### DISCUSSION TITLE
Is sqlite3 required for installation?

### DISCUSSION THREAD

**Discussion by oma-s** [avo-hq/avo#654] [Help] [answered]
Hi there!

We are using postgreSQL not only in production but also in test and development. So the sqlite gem is not in the gem file. When I first tried to install 'avo' I got an error because of the missing sqlite3 adapter. I could install it by putting sqlite in the gem file. Is there any other way around this?

Many thanks and best regards!

**Comment by adrianthedev** [comment, marked as answer]
Hey @oma-s. `sqlite` shouldn't be required for Avo to work. I use it in a few apps without the `sqlite` gem. I also use PostgreSQL in development.

Is this a new app or an existing one? Did this happen when you tried to install just Avo or a few other packages?

Can you paste the error output here? Maybe we can troubleshoot it together?

**Reply by oma-s** [reply]
@adrianthedev Thanks for the quick reply. I only added this one gem to an existing app. I will try to reproduce the error and get back to you with the output :)

**Reply by adrianthedev** [reply]
Sounds good. Thanks!

**Reply by oma-s** [reply]
Actually I tried reproducing the behaviour, but I could not. Seems it was an issue on my machine. :)  Sorry, I guess we can close the issue. Thanks for the quick reply!
